stories/format.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Style/Format" />
+
+# Format
+
+## Number format
+
+- Use of [space] for [internationally recommended](https://en.wikipedia.org/wiki/Decimal_separator) thousands separators. No comma or point, as these are reserved for use as the decimal sign.
+- Numbers whose magnitude is less than 1, the decimal sign should be preceded by a zero.
+
+## Date format
+
+### Code Description
+
+| Code | Description                                | Example |
+| ---- | ------------------------------------------ | ------- |
+| YYYY | Year on 4 digits                           | 2021    |
+| EEE  | Weekday on 3 letters, first with uppercase | "Mon"   |
+| DD   | Day of the month with leading zero         | 08      |
+| MM   | Numerical month with leading zero          | 07      |
+| MMM  | Month abbreviation on 3 letters uppercase  | "JUL"   |
+| HH   | Hour [0-23]                                | 13      |
+| mm   | Minute with leading zero                   | 05      |
+| ss   | Second with leading zero                   | 06      |
+
+### Format list
+
+| Alias   | code                     | Example                  | Length | Context                                              |
+| ------- | ------------------------ | ------------------------ | ------ | ---------------------------------------------------- |
+| Short#1 | DD MMM                   | 20 Jul                   | 5      | Chart time axis                                      |
+| Short#2 | DDMMM HH:mm              | 20JUL 09:00              | 11     | Limited space - Dense tables, charts                 |
```

Comment:
Is it really the one we want to use in dense tables? I was thinking the MMM was causing some alignment issues and that it was preferable to use numeric format?

Reply:
Correct, I change the "table" use cases.